### PR TITLE
Add kubeedge API

### PR DESCRIPTION
**Pull request by Ma-Dan**
Signed-off-by: root <danma@yunify.com>

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Add KubeEdge API support to console for edge functions

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/simple/client/kubeedge/options.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+func (o *Options) Validate() []error {
+	errs := []error{}
+
+	return errs
+}
+
+func (o *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
+	fs.StringVar(&o.Endpoint, "edge-server-endpoint", c.Endpoint,
```
Comment (benjaminhuo) [inline pkg/simple/client/kubeedge/options.go:45]: edge-server-endpoint => edge-watcher-endpoint

File: pkg/simple/client/kubeedge/options.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		"edge server endpoint for edge v1alpha1.")
```
Comment (benjaminhuo) [inline pkg/simple/client/kubeedge/options.go:46]: edge watcher endpoint.

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: pkg/simple/client/kubeedge/options.go (hunk 1 above)
Comment (Ma-Dan) [reply pkg/simple/client/kubeedge/options.go]: Fixed

File: pkg/simple/client/kubeedge/options.go (hunk 2 above)
Comment (Ma-Dan) [reply pkg/simple/client/kubeedge/options.go]: Fixed

**Review by benjaminhuo — APPROVED** [review]

**Comment by zryfish** [comment]
@Ma-Dan conflict, please help reslove first

**Comment by Ma-Dan** [comment]
> @Ma-Dan conflict, please help reslove first

Rebased

**Comment by zryfish** [comment]
CI failed, please use a real kubeedge options @Ma-Dan

**Pull request merged**